doc/developing_extensions/topics/developing_extensions.adoc

```diff
@@ -908,9 +908,9 @@ public class AWSS3DeleteObjectCustomizer implements ComponentProxyCustomizer {
 +
 Defines the class implementing https://github.com/syndesisio/syndesis/blob/master/app/integration/component-proxy/src/main/java/io/syndesis/integration/component/proxy/ComponentProxyFactory.java[ComponentProxyFactory] used to create and/or configure the underling component/endpoint.  It can be set on _action_ or global level with the _action_ one having precedence.
 +
-To customize the behavior of the https://github.com/syndesisio/syndesis/blob/master/app/integration/component-proxy/src/main/java/io/syndesis/integration/component/proxy/ComponentProxyComponent.java[ComponentProxyComponent] you can override on of the following methods:
+To customize the behavior of the https://github.com/syndesisio/syndesis/blob/master/app/integration/component-proxy/src/main/java/io/syndesis/integration/component/proxy/ComponentProxyComponent.java[ComponentProxyComponent] you can override the following methods:
```

Comment:
Is this paragraph supposed to be a sub paragraph of the paragraph that describes connectorFactory? I'm wondering whether this paragraph should not be indented at all. It should have its own heading, something like, "Customizing behavior of ComponentProxyComponent" and then you do not need to change the levels of the bullets. But I may be wrong.  If I'm right, then you just need to remove the plus sign that is on line 910. And then the single-asterisk indents will be correct.